src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -126,20 +132,28 @@ def load_mlflow_model(
         model = mlflow.pyfunc.load_model(model_uri)._model_impl
         return model
     except Exception as e:
-        raise ValueError(
+        raise UserConfigError(
             "Unable to load mlflow model from {} in current environment due to error:\n{}".format(
                 model_uri, e
-            )
+            ), e
         )
 
 
 def _classify_and_log_pip_install_error(elog):
-    if elog is not None:
-        if b"Could not find a version that satisfies the requirement" in elog:
-            _logger.warning("Detected unsatisfiable version requirment.")
+    ret_message = []
+    if elog is None:
+        return ret_message
+
+    if "Could not find a version that satisfies the requirement" in elog:
```

Comment:
the output of subprocess.CalledProcessError is bytes-like object, not a str. So if we want to check if certain string is in the output, we need to convert str to bytes-like object.
Details can be found here: https://github.com/Azure/RAI-vNext-Preview/pull/180

Reply:
thanks! fixed.